include/hip/hcc_detail/device_library_decls.h

```diff
@@ -58,6 +58,15 @@ extern "C" __device__ __attribute__((const)) float __ocml_trunc_f32(float);
 extern "C" __device__ __attribute__((const)) float __ocml_fmin_f32(float, float);
 extern "C" __device__ __attribute__((const)) float __ocml_fmax_f32(float, float);
 
+extern "C" __device__ __attribute__((const)) uint32_t __ockl_lane_u32();
```

Comment:
lane, both syncs and is_valid should only get convergent attribute.

Reply:
That's not strictly necessary as convergent is assumed by default